### PR TITLE
Initial clone to public

### PR DESCRIPTION
**Pull request by jrit**
@Tathanen just copied and updated readme. Thoughts?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
...
+
+## Use
+
+To initialize, pass a URL with the token `{page}` which will be replaced as each page is loaded.
+
+```js
+$scope.list = new infiniteScroll( "/users?p={page}" );
+```
+
+The attributes shown here are attributes of ngInfiniteScroll, but the conditionals and callbacks are part of infiniteScrollDecorator.
```
Comment (Tathanen) [inline README.md]: So ngInfiniteScroll is an external library?  I guess that wasn't clear to me from the start, maybe put a link to it somewhere in the readme, perhaps just link the reference in that first sentence at the top.  (This is my first time ever looking at this really.)

Would be nice if the "conditionals and callbacks" were actually documented/explained in any capacity.

Comment (jrit) [inline README.md]: Agreed

**Comment by jrit** [comment]
Have another look at docs

**Comment by Tathanen** [comment]
Better!

Do you wanna match the format (params, types, etc.) from the other readmes tho?  See: https://github.com/vokal/vokal-ng-api

**Comment by jrit** [comment]
Looks more like that now

File: README.md [hunk 2]
```diff
@@ -1,2 +1,96 @@
...
+
+*String* | Default: set by argument during initialization
+
+The paging url
+
+* * *
+
+#### `API`
+
+*Service* | Default: set by argument during initialization
```
Comment (Tathanen) [inline README.md:39]: I'm not sure `Service` is a JavaScript type.

Comment (jrit) [inline README.md:39]: I feel like specifying any type is ok, though I'll move the info about the type down from just below the Initialization header

**Comment by Tathanen** [comment]
Good enough!!!

**Pull request merged**